cockpit-composer.spec.in

```diff
@@ -31,6 +31,7 @@ the cloud. It integrates into Cockpit as a frontend for osbuild.
 mkdir -p %{buildroot}/%{_datadir}/cockpit/composer
 cp -a public/* %{buildroot}/%{_datadir}/cockpit/composer
 mkdir -p %{buildroot}/%{_datadir}/metainfo/
+appstream-util validate-relax --nonet public/io.weldr.cockpit-composer.metainfo.xml
```

Comment:
You need a new `BuildRequires: libappstream-glib` for that. It works in our CI as that is pre-installed into the mocks, but it'll fail in koji/brew.